src/pages/import/components/ReplaceTimetable.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState, useEffect } from 'react';
+
+import {
+  useToast,
+  useDisclosure,
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  ModalBody,
+  ModalFooter,
+  Text,
+  VStack,
+  Alert,
+  AlertIcon,
+  useMediaQuery,
+} from '@chakra-ui/react';
+import { BsArrowRepeat } from 'react-icons/bs';
+
+import { Timetable } from 'lib/fetchers';
+import { SavedCourse, useSavedCourses } from 'lib/hooks/useSavedCourses';
+
+import { SelectedCoursesCardList } from 'pages/scheduler/components/share/SelectedCoursesCardList';
+
+export function ReplaceTimetable({ loading, data }: { loading: boolean; data: Timetable }): JSX.Element {
+  const { replaceCourses } = useSavedCourses();
+  const toast = useToast();
+  const [isSmallScreen] = useMediaQuery('(min-width:680px)');
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const filteredCoursesList: SavedCourse[] = [];
+  const [filteredCourses, setFilteredCourses] = useState(filteredCoursesList);
+  const [timetableTerm, setTimetableTerm] = useState('');
+
+  const handleReplace = () => {
+    replaceCourses(data.courses, data.term);
+    onClose();
+    toast({
+      title: 'Timetable Replaced!',
```

Comment:
```suggestion
      title: 'Timetable replaced!',
```
Also same thing, maybe we could just navigate directly to the user's timetable after replacing